Formula/px4-sim-gazebo.rb

```diff
@@ -12,28 +12,28 @@ def message
 end
 
 class Px4SimGazebo < Formula
-  desc "PX4 Gazebo simulation"
+  desc "PX4 Gazebo Simulation Toolkit"
   homepage "http://px4.io"
-  url "https://raw.githubusercontent.com/PX4/PX4-Autopilot/master/Tools/px4.py"
-  version "1.11.0"
-  sha256 "6e4a8a6aad592a0c156d16233e16ee276fd6cc63b620be5bd047375b1352e27d"
+  url "https://raw.githubusercontent.com/PX4/PX4-Autopilot/main/Tools/px4.py"
+  version "1.15.0"
+  sha256 "5eda2111dc20c092dc8241768121cf4173edabb593b2b199e6c233b5088c52aa"
   depends_on "exiftool"
   depends_on "glog"
   depends_on "graphviz"
-  depends_on "gst-libav"
-  depends_on "gst-plugins-bad"
-  depends_on "gst-plugins-base"
-  depends_on "gst-plugins-good"
-  depends_on "gst-plugins-ugly"
   depends_on "gstreamer"
   depends_on "opencv"
-  depends_on "osrf/simulation/gazebo11"
+  depends_on "osrf/simulation/gz-harmonic"
   depends_on "protobuf"
   depends_on "px4-dev"
 
   def install
     mkdir_p "#{bin}/"
     cp "px4.py", "#{bin}/px4-sim-gazebo.py"
-    ohai "PX4 Gazebo simulation installed"
+    ohai "PX4 Gazebo Simulation Installed"
```

Comment:
@mrpollo How does this affect the macOS setup instructions for normal users - as documented in https://docs.px4.io/main/en/dev_setup/dev_env_mac.html ?

My assumption is that this allows "homebrew" to install the toolchain using commands like this:

brew tap PX4/px4
brew install px4-dev
brew install px4-sim-gazebo

Not clear if you still need to update PIP or if this works on all silicon ???

Can you update those docs - I've not got (or ever used) a mac